src/Symfony/Component/HttpFoundation/File/File.php

```diff
@@ -133,4 +133,18 @@ protected function getName($name)
 
         return $originalName;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getSize()
+    {
+        // Get the file size from a stream rather than from disk in case it is changed later by
+        // stream wrappers or stream filters.
+        $file = fopen($this->getPathname(), 'rb');
+        $fstat = fstat($file);
```

Comment:
ok, just read your linked issue.
any way to add a test case that would break when using the parent getSize()?